bob/participant-info.json

```diff
@@ -43,29 +43,21 @@
       "title": "EMV Participant Information",
       "type": "object",
       "required": [
-        "iin"
+        "aid"
       ],
       "properties": {
-        "iin": {
-          "title": "Issuer Identification Numbers",
-          "type": "array",
-          "items": {
-            "type": "integer",
-            "example": 97523124
-          }
+        "aid": {
+          "title": "Application Identifier",
+          "type": "string",
```

Comment:
Change to "array" to allow multiple registred AID:s e.g. "RID + PIX1", "RID + PIX2"

["A0000008381010", "A0000008381011"]

Reply:
9ed113e861469b3de5c8037da7235bea3eabefe4